lib/aoc.ex

```diff
@@ -229,6 +229,14 @@ defmodule AOC do
   @spec example_stream(pos_integer(), pos_integer()) :: Enumerable.t()
   def example_stream(year, day), do: example_path(year, day) |> path_to_stream()
 
-  defp path_to_string(path), do: path |> File.read!() |> String.trim_trailing()
-  defp path_to_stream(path), do: path |> File.stream!() |> Stream.map(&String.trim/1)
+  defp path_to_string(path) do
+    path
+    |> File.read!
```

Comment:
There is no need to remove the trim here, as it only remove the trailing newline at the end, which should never pose an issue.